File: src/targets/node/request/fixtures/cookies.js
const request = require('request');

const jar = request.jar();
<<<<<<< HEAD
jar.setCookie(request.cookie('foo=bar'), 'https://httpbin.org/cookies');
jar.setCookie(request.cookie('bar=baz'), 'https://httpbin.org/cookies');

const options = {method: 'GET', url: 'https://httpbin.org/cookies', jar: jar};
=======
jar.setCookie(request.cookie('foo=bar'), 'http://mockbin.com/har');
jar.setCookie(request.cookie('bar=baz'), 'http://mockbin.com/har');

const options = {method: 'POST', url: 'http://mockbin.com/har', jar: jar};
>>>>>>> upstream/master

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});

File: src/targets/java/nethttp/fixtures/cookies.java
HttpRequest request = HttpRequest.newBuilder()
<<<<<<< HEAD
    .uri(URI.create("https://httpbin.org/cookies"))
    .header("cookie", "foo=bar; bar=baz")
    .method("GET", HttpRequest.BodyPublishers.noBody())
=======
    .uri(URI.create("https://httpbin.org/anything"))
    .header("content-type", "application/json")
    .method("POST", HttpRequest.BodyPublishers.noBody())
>>>>>>> upstream/master
    .build();
HttpResponse<String> response = HttpClient.newHttpClient().send(request, HttpResponse.BodyHandlers.ofString());
System.out.println(response.body());